backend/src/rss.rs

```diff
@@ -122,9 +148,11 @@ async fn video_items(
         let event_link = format!("{tobira_url}/!v/{tobira_event_id}");
         let thumbnail = &event.thumbnail_url.unwrap_or_default();
         let tracks = preferred_tracks(event.tracks);
-        let enclosure_track = tracks.1.unwrap();
+        let enclosure_track = tracks.0.unwrap();
         let enclosure_url = &enclosure_track.uri;
```

Comment:
Ah, same with this. No need for a variable I'd think. Less code is always good and when reading the XML, there is one step fewer to understand what value is interpolated there.